.env.example

```diff
@@ -4,3 +4,4 @@ DEFENDER_API_KEY=<OPENZEPPELIN_DEFENDER_API_KEY>
 DEFENDER_API_SECRET=<OPENSEPPELIN_DEFENDER_API_SECRET>
 SECRET_NAMES=projects/1027349420744/secrets/funding-layer/versions/1
 ALCHEMY_KEY=<ALCHEMY_KEY>
+HYPERSYNC_API_KEY=<https://docs.envio.dev/docs/HyperSync/api-tokens#generating-api-tokens>
```

Comment:
Can we add this to the GCP `funding-layer` secret?

Reply:
Added!